charts/alfresco-share/templates/deployment.yaml

```diff
@@ -19,22 +19,35 @@ spec:
   template:
     metadata:
       annotations:
-        {{- if not .Values.repository.existingConfigMap }}
-        checksum/config: {{ include (print $.Template.BasePath "/config-share.yaml") . | sha256sum }}
+        {{- with .Values.podAnnotations }}
+          {{- toYaml . | nindent 8 }}
         {{- end }}
       labels:
         {{- include "alfresco-share.selectorLabels" . | nindent 8 }}
+        {{- with .Values.podLabels }}
+          {{- toYaml . | nindent 8 }}
+        {{- end }}
     spec:
       serviceAccountName: {{ include "alfresco-share.serviceAccountName" . }}
       {{- include "component-pod-security-context" .Values | indent 4 }}
-      {{- include "alfresco-content-services.imagePullSecrets" . | indent 6 }}
+      {{- include "alfresco-common.imagePullSecrets" . | indent 6 }}
+      {{- with .Values.nodeSelector }}
       nodeSelector:
-        {{- toYaml .Values.nodeSelector | nindent 8 }}
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.affinity }}
+      affinity:
+        {{- tpl . $ | nindent 8 }}
+      {{- end }}
+      {{- with .Values.tolerations }}
+      tolerations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
```

Comment:
I know it's not top prio, but I started to follow the resource ordering of the helm boilerplate when creating a new chart

see https://github.com/gionn/helm-new-chart/blob/master/templates/deployment.yaml#L57

those are defined at the bottom of the deployment